### DISCUSSION TITLE
Add a brightness adjustment?

### DISCUSSION THREAD

**Discussion by brunoosti** [AutoDarkMode/Windows-Auto-Night-Mode#632] [Q&A]
Besides switching to dark mode, it would be nice to have an option for dimming the monitor's brightness. 
As an example: I work facing a window, so my monitor's brightness is cracked up to balance sunlight. During night hours, though, it tends to be uncomfortable. 
Thank you for the great app! : )

**Comment by Armin2208** [comment]
You can do this by yourself with [Auto Dark Mode scripts](https://github.com/AutoDarkMode/Windows-Auto-Night-Mode/wiki/How-to-add-custom-scripts) and [Twinkle Tray which supports command line arguments](https://github.com/xanderfrangos/twinkle-tray#command-line-arguments).
We are not planing to implement this into Auto Dark Mode core.

**Reply by Spiritreader** [reply]
That being said, if anyone decides to write a script config for this, it would be a great contribution to the Auto Dark Mode community if they made a post in our scripts discussion section.
Mainly a manual on how to set it up so others can also make use of it.
